cfme/networks/provider/__init__.py

```diff
@@ -272,7 +272,7 @@ def delete(self, cancel=True):
         """
         view = navigate_to(self, 'Details')
         view.toolbar.configuration.item_select('Remove this Network Provider from Inventory',
-                                               handle_alert=not cancel)
+                                               handle_alert=True)
```

Comment:
I don't think this is a good change - in `item_select`, passing `True` here means the alert will always be confirmed.

Which means that the 'cancel' boolean no longer works in this method.

Reply:
Removed this change